app/services/migration/migrators/user.rb

```diff
@@ -44,8 +44,16 @@ def call
 
   private
 
-    def validate_multiple_trns!(unique_trns, user)
-      return unless unique_trns.size > 1
+    def unique_validated_trns(ecf_user)
+      ecf_user.npq_applications
+        .where(teacher_reference_number_verified: true)
```

Comment:
maybe we use select to avoid an extra query here 🤔

Reply:
As we `pluck` the query results I don't think `select` would add any benefit?

Reply:
so initially I wanted to avoid a query altogether and do it in memory and add a select loop but since it has to be a query we can't get around it I guess